streams/src/main/java/org/apache/kafka/streams/kstream/UnlimitedWindows.java

```diff
@@ -93,4 +93,8 @@ public int hashCode() {
         return (int) (start ^ (start >>> 32));
     }
 
+    @Override
+    public long maintainMs() {
```

Comment:
Unless of course you are referring to RocksDBSegments in which there should only be 1 for UnlimitedWindows - right? Whenever we create a new window we use `windowStartMs`, which for an unlimited window this is always fixed.